docs/PLUGIN_CATALOG.md

```diff
@@ -92,5 +94,4 @@ There will always be more plugins we wish we had. To make sure others can contri
 | Collectd (native) - [Issue needed](https://github.com/intelsdi-x/snap/issues/)        | Collector | Compatibility for legacy environments |                                                                                          |
 | Prometheus    - [#1196](https://github.com/intelsdi-x/snap/issues/1196)               | Publisher | As part of a CNCF stack               |                                                                                          |
 | Kubernetes Node (Minion) - [Issue needed](https://github.com/intelsdi-x/snap/issues/) | Collector | Monitoring nodes in k8s               |                                                                                          |
-| JVM (via JMX) - [Issue needed](https://github.com/intelsdi-x/snap/issues/)            | Collector |                                       |                                                                                          |
 | Ceilometer - [Issue needed](https://github.com/intelsdi-x/snap/issues/)               | Publisher | possibly just OSLO proc + RMQ         |                                                                                          |
```

Comment:
I was thinking by plugin type {collector->process->publisher} and within those groups could be alphabetical.